src/pages/create-course/CreateCourse.tsx

```diff
@@ -94,6 +94,17 @@ const CreateCourse = () => {
     handleNonInputValueChange('sections', value)
   }
 
+  const handleSectionResourcesOrder = (
+    id: string,
+    resources: CourseResources[]
+  ) => {
+    const sectionToEdit = data.sections.find((section) => section.id === id)
+    const orderedResources = resources.map((resource) => resource._id)
+    if (sectionToEdit) {
```

Comment:
You should move the resources map under the if statement. Cause there is no sense to map it if no item was found.

Reply:
moved